sample/deterministic_sampler_test.go

```diff
@@ -0,0 +1,84 @@
+package sample
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+)
+
+const requestIDBytes = `abcdef0123456789`
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+func randomRequestID() string {
+	// create request ID roughly resembling something you would get from
+	// AWS ALB, e.g.,
+	//
+	// 1-5ababc0a-4df707925c1681932ea22a20
+	//
+	// The AWS docs say the middle bit is "time in seconds since epoch",
+	// (implying base 10) but the above represents an actual Root= ID from
+	// an ALB access log, so... yeah.
```

Comment:
5ababc0a is hex for 1522187274 which is 2018-03-27, so the docs aren't wrong, just confusing. :)

Reply:
ah, i was wondering about that. good call out